### ISSUE TITLE
Widget light service call not working

### ISSUE THREAD

**Issue by Gronsak** [home-assistant/home-assistant-android#320]
**Home Assistant Android version:** 
1.5.0
**Android version:** 
10
**Phone model:** 
Samsung Galaxy S10 (SM-G973F)
**Home Assistant version:** 
0.104.3
**Description of problem:**
Light.toggle/turn_on/turn_off seems to have a problem calling my server, I get a webhook error corresponding to when I'm pressing the widget.
Widget icon responds as if successful with green checkbox.
**Additional information:**

From Home Assistant log:
```
2020-01-29 23:12:02 WARNING (MainThread) [homeassistant.components.webhook] Received message for unregistered webhook null
```

**Comment by JBassett** [comment]
Try cleaning app data and logging back in.  It looks like the webhook id isn't saved on device anymore.

**Comment by Gronsak** [comment]
Yep that fixed it, fells like a some form of check for that is in order, no indication on what went wrong on the app side only the server log

**Comment by KBerstene** [comment]
I think the main problem with implementing a check in the app is that the response to a REST service call is always 200 as long as the call itself is received.  HA doesn't supply any sort of data saying that the service data is bad--it just passes it along to the appropriate service and tries to execute it there.

I think this is a change that would have to first happen in HA to send the app a response that indicates that the service call failed somewhere down the line.

**Comment by balloob** [comment]
Fixed in #370
